generators.py

```diff
@@ -0,0 +1,21 @@
+# fill in this function
+def fib():
```

Comment:
```suggestion
def fib() -> Generator[int, None, None]:
```
Python Generators should be typed in such a way that they [define the type of the the variable they yield,  the type of the variable they expect to receive (using `.send()`) and the types of the variables they eventually return.](https://docs.python.org/3/library/typing.html#corresponding-to-other-types-in-collections-abc)

In this case, since this Generator does not receive any value through `.send()` nor does it return a value, the other two types must thus be set to `None`

Reply:
Thanks Rizwan bhai, will keep that in mind for next time. solved